apps/helper-app/src/home/home.component.scss

```diff
@@ -15,11 +15,10 @@ input[type='number'] {
 .header {
   width: 100%;
   position: relative;
-  height: 400px;
+  height: 500px;
```

Comment:
It's too high for the image which causes a gap between the top of the window and the image:

![image](https://user-images.githubusercontent.com/12248527/77767170-ef1b8f00-7040-11ea-8498-2319a24c3a82.png)

**Possible solutions:**
- Decrease height attribute
- Change `background-position` from `center` to `top`

Reply:
same if you open it on desktop